packages/core/src/app/payment/storedInstrument/isInstrumentCardNumberRequired.ts

```diff
@@ -13,5 +14,13 @@ export default function isInstrumentCardNumberRequired({
         return false;
     }
 
-    return !instrument.trustedShippingAddress;
+    if (instrument.trustedShippingAddress) {
+        return false;
+    } else {
+        if (instrument.untrustedShippingCardVerificationMode === UntrustedShippingCardVerificationType.CVV) {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
```suggestion
    if (instrument.trustedShippingAddress) {
        return false;
    }  
    
    return instrument.untrustedShippingCardVerificationMode === UntrustedShippingCardVerificationType.CVV ? false : true;
```

Reply:
```
instrument.untrustedShippingCardVerificationMode === UntrustedShippingCardVerificationType.CVV ? false : true;
```
Lint error will be raised, so I have updated it to 
```
return !(instrument.untrustedShippingCardVerificationMode === UntrustedShippingCardVerificationType.CVV)
```